deploy/cluster_role_che.yaml

```diff
@@ -0,0 +1,142 @@
+#
+#  Copyright (c) 2012-2019 Red Hat, Inc.
+#    This program and the accompanying materials are made
+#    available under the terms of the Eclipse Public License 2.0
+#    which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+#  SPDX-License-Identifier: EPL-2.0
+#
+#  Contributors:
+#    Red Hat, Inc. - initial API and implementation
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: che
+  labels:
+    app: che
+    component: che
+rules:
+  - verbs:
+      - get
+      - create
+    apiGroups:
+      - authorization.openshift.io
+      - rbac.authorization.k8s.io
+    resources:
+      - roles
+  - verbs:
+      - get
+      - update
+      - create
+    apiGroups:
+      - authorization.openshift.io
+      - rbac.authorization.k8s.io
+    resources:
+      - rolebindings
+  - verbs:
+      - get
+    apiGroups:
+      - project.openshift.io
+    resources:
+      - projects
+  - verbs:
+      - get
+      - create
+    apiGroups:
+      - ''
+    resources:
+      - serviceaccounts
+  - verbs:
+      - create
+    apiGroups:
+      - ''
+    resources:
+      - pods/exec
+  - verbs:
+      - list
+    apiGroups:
+      - ''
+    resources:
+      - persistentvolumeclaims
+      - configmaps
+  - verbs:
+      - list
+    apiGroups:
+      - apps
+    resources:
+      - secrets
+  - verbs:
+      - list
+      - create
+      - delete
+    apiGroups:
+      - ''
+    resources:
+      - secrets
```

Comment:
Could you elaborate on why we have two `secrets` with different `apiGroups` and `verbs`?